deep_qa/training/run_model.py

```diff
@@ -0,0 +1,117 @@
+
```

Comment:
Extra blank line at the top.  Also, what do you think about calling this `run.py`, instead of `run_model.py`?